src/modules/comment.js

```diff
@@ -0,0 +1,152 @@
+        <p class="modal-genres">${getGenreList(movie)}</p>
+        <div class="modal-bottom-div">
+          <div class="flex modal-likes">
+                      <button type="button" class="likeBtn" id="${movie.id}">
+                        <svg
+                          xmlns="http://www.w3.org/2000/svg"
+                          fill="none"
+                          viewBox="0 0 24 24"
+                          stroke-width="1.5"
+                          stroke="currentColor"
+                          class="w-6 h-6"
+                        >
+                          <path
+                            stroke-linecap="round"
+                            stroke-linejoin="round"
+                            d="M21 8.25c0-2.485-2.099-4.5-4.688-4.5-1.935 0-3.597 1.126-4.312 2.733-.715-1.607-2.377-2.733-4.313-2.733C5.1 3.75 3 5.765 3 8.25c0 7.22 9 12 9 12s9-4.78 9-12z"
+                          />
+                        </svg>
+                      </button>
+                      <span>${movie.likes ? movie.likes : 0} likes</span>
+                    </div>
+        </div>
+      </div>
+    </div>
+    <hr>
+    <div class="modal-comments" id="modal-comments">
+      <div class="comments-header">
+        <h2>Comments</h2><span class="comments-count" id="modal-commets-count"></span>
+      </div>
+      <ul class="comments-list">
+        <li class="comment-item">
+          <span class="comment-time">15/03/2023</span
+          ><span class="commenter">Dare:</span
+          ><span class="comment-body"></span>
+        </li>
+      </ul>
+    </div>
+    <hr>
+    <div class="modal-new-comment" id="modal-comment-add">
+      <h2>Add a comment</h2>
+      <form id="form-comment">
+        <input
+          class="form-control form-control-lg"
+          type="text" name="commenter"
+          placeholder="Your name"
+        />
+        <textarea
+          class="form-control form-control-lg"
+          id="exampleFormControlTextarea1"
+          name="comment"          rows="3"
+          placeholder="Your insights"
+        ></textarea>
+        <button type="submit" id="submit-comment" class="btn btn-danger">Comment</button>
+      </form>
+    </div>
+    `;
+
+const commentItem = (comment) => {
+  const listItem = document.createElement('li');
+  listItem.className = 'comment-item';
+
+  listItem.innerHTML = `
+          <span class="comment-time">${comment.creation_date}</span>
+          <span class="commenter">${comment.username}:</span>
+          <span class="comment-body">${comment.comment}</span>
+    `;
+
+  return listItem;
+};
+
+const displayComments = (modalContainer, movieId) => {
+  const commentsContainer = modalContainer.querySelector('.modal-comments ul');
+  const commentCountView = modalContainer.querySelector('#modal-commets-count');
+  commentsContainer.innerHTML = '';
+  invovlementapi.getMovieComments(movieId).then((comments) => {
+    commentCountView.innerHTML = comments.length ?? 0;
```

Comment:
Hello @adamilare 
Good job implementing the counter for the comment. I suggest you use a separate module for implementing the counter for the comments as stated in issue #6. The counter function should be implemented using the elements instead of the data array. Please refer to the issue for more details